src/megatron/bridge/training/config.py

```diff
@@ -164,6 +164,10 @@ class DataloaderConfig:
 @dataclass
 class GPTDatasetConfig(MCoreGPTDatasetConfig, DataloaderConfig):
     """Configuration specific to GPT datasets, inheriting from MCore and base DataloaderConfig."""
+    
+    get_attention_mask_from_fusion: bool = False
+    """If set, the dataset will not pass a None attention mask and the casual 
+    mask is autogenerated from the attn backend"""
```

Comment:
`will not pass` or `will pass`?
Also, I think `casual mask` should be just `attention mask` because casual is an attention type?